nodestream/pipeline/extractors/files.py

```diff
@@ -58,7 +58,7 @@ def read_file_from_handle(self, fp: StringIO) -> Iterable[JsonLikeDocument]:
 
 class JsonFileFormat(SupportedFileFormat, alias=".json"):
     def read_file_from_handle(self, fp: StringIO) -> Iterable[JsonLikeDocument]:
-        return [json.load(fp)]
+        return [json.loads(fp)]
```

Comment:
This class method expects a string as input, so it must use json.loads()

Reply:
This does not expect a string as input, it expects a `StringIO`.